code/modules/client/verbs/ooc.dm

```diff
@@ -100,9 +97,12 @@ GLOBAL_VAR_INIT(normal_ooc_colour, OOC_COLOR)
 	set name = "Set Your OOC Color"
 	set category = "Preferences"
 
+	/*
+	Not sure how analy retarded you made this exactly TG.
```

Comment:
Remie, you got it backwards.
Admins set the entire string color, members set the name color.